subsys/bluetooth/mesh/transport.c

```diff
@@ -632,12 +632,12 @@ int bt_mesh_trans_send(struct bt_mesh_net_tx *tx, struct net_buf_simple *msg,
 		return -EINVAL;
 	}
 
-	if (msg->len > BT_MESH_TX_SDU_MAX) {
-		BT_ERR("Not enough segment buffers for length %u", msg->len);
+	if (msg->len > BT_MESH_TX_SDU_MAX - BT_MESH_MIC_SHORT) {
```

Comment:
This looks unrelated to the reword of the timer, is this a bug-fix?

Should be it's own commit if it is.

Reply:
Split into its own commit.